discord-deleter.go

```diff
@@ -40,14 +40,15 @@ func main() {
 func OnMessageCreate(s *discordgo.Session, m *discordgo.MessageCreate, commands []commands.ICommandHandler) {
 	var cmdSlices = strings.Split(strings.ToLower(m.Content), "")
 
+	// Iterate through command handlers till it finds one that is able to handle command
```

Comment:
I'm not sure where the right place is to make comments like these, so I'm just going to put it here.

Regular file names should be lower case, short (single word), and without any underscores or spaces. You should take a look at [this](https://golang.org/doc/effective_go#package-names).

Reply:
On this note, I suggest renaming these files too:
* commands/ICommandHandler.go -> commands/handler.go
* commands/MessageCollector.go -> commands/message.go OR collector.go (or specialized names if you want one file per collector)

Reply:
Refactored it as such, the command handler (in handler.go), is a core part of the app. So moved it to root directory/namespace. Things like Message collector are the individual commands people would implement so kept those under the commands/ namespace.